cypress/e2e/undo-redo/block-configs-should-update-flow-state/content--blocks-configs.cy.js

```diff
@@ -0,0 +1,90 @@
+describe('mutate flow state for CONTENT blocks', () => {
+  beforeEach(() => {
+    cy.createFlow({
+      label: 'adding blocks test',
+    })
+  })
+
+  it('should validate Location Response Block', () => {
+    const blockConfigs = {
+      accuracy: {
+        threshold: 25,
+        timeout: 134
+      },
+    }
+    const getStore = () => cy.window().its('store')
+    const flowModuleState = () => getStore().its('state.flow')
+    const flowsListState = () => flowModuleState().its('flows')
+
+    cy.addBlock(['Location Response'])
+    const firstBlockState = () => flowsListState().its('[0].blocks[0]')
+
+    cy.get('[data-cy="accuracy-threshold--editor"]')
+      .find('input')
+      .clear()
+      .type(blockConfigs.accuracy.threshold)
+    firstBlockState().its('config.accuracy_threshold_meters').should('equal', blockConfigs.accuracy.threshold)
+
+    cy.get('[data-cy="accuracy-timeout--editor"]')
+      .find('input')
+      .clear()
+      .type(blockConfigs.accuracy.timeout)
+    firstBlockState().its('config.accuracy_timeout_seconds').should('eq', blockConfigs.accuracy.timeout)
+  })
+
+  it('should validate Numeric Response Block', () => {
+    const blockConfigs = {
+      minimum: 13,
+      maximum: 950,
+      maxiDigit: 4
+    }
+    const getStore = () => cy.window().its('store')
+    const flowModuleState = () => getStore().its('state.flow')
+    const flowsListState = () => flowModuleState().its('flows')
+
+    cy.addBlock(['Numeric Response'])
+    const firstBlockState = () => flowsListState().its('[0].blocks[0]')
+
+    // ######## Num values
+    cy.get('[data-cy="minimum-numeric--editor"]')
+      .find('input')
+      .clear()
```

Comment:
@pascaleli , we can "clear" the input first to avoid appending new typed characters into it.

cc: @bulatgab , @object-required